spec/features/spo_handover_cases_feature_spec.rb

```diff
@@ -67,71 +68,106 @@
-  end
-
   context 'with four offenders' do
-    let(:poms) { ["Dunlop, Abbey", "Brown, Denis", "Albright, Sally", "Carsley, Jo"] }
+    let(:pom_names) { ["Dunlop, Abbey", "Brown, Denis", "Albright, Sally", "Carsley, Jo"] }
     let(:coms) { ["Dabery, Suzzie", "Canne, Sam", "Blackburn, Zoe", "Abbot, Brian"] }
 
     let(:check_poms) { ["Albright, Sally", "Brown, Denis", "Carsley, Jo", "Dunlop, Abbey"] }
     let(:check_coms) { ["Abbot, Brian", "Blackburn, Zoe", "Canne, Sam", "Dabery, Suzzie"] }
+    let(:check_dates) { ["Abbot, Brian", "Canne, Sam", "Blackburn, Zoe", "Dabery, Suzzie"] }
 
     before do
       stub_auth_token
-      stub_user(staff_id: 123_456)
+      stub_user(staff_id: user.staff_id)
 
       offenders = [
           build(:nomis_offender,
                 offenderNo: "A7514GW",
-                sentence: attributes_for(:sentence_detail, :inside_handover_window)),
+                sentence: attributes_for(:sentence_detail, :handover_in_8_days)),
           build(:nomis_offender,
                 offenderNo: "B7514GW",
-                sentence: attributes_for(:sentence_detail, :inside_handover_window)),
+                sentence: attributes_for(:sentence_detail, :handover_in_4_days)),
           build(:nomis_offender, offenderNo: "C7514GW",
-                sentence: attributes_for(:sentence_detail, :inside_handover_window)),
+                sentence: attributes_for(:sentence_detail, :handover_in_6_days)),
           build(:nomis_offender, offenderNo: "D7514GW",
-                sentence: attributes_for(:sentence_detail, :inside_handover_window))
+                sentence: attributes_for(:sentence_detail, :handover_in_3_days))
       ]
 
       stub_offenders_for_prison(prison, offenders)
 
       offenders.each_with_index do |offender, i|
         create(:case_information,  com_name: coms.fetch(i),  nomis_offender_id: offender.fetch(:offenderNo))
-        create(:allocation, primary_pom_name: poms.fetch(i), nomis_offender_id: offender.fetch(:offenderNo), prison: prison)
+        create(:allocation, primary_pom_nomis_id: user.staff_id, primary_pom_name: pom_names.fetch(i), nomis_offender_id: offender.fetch(:offenderNo), prison: prison)
       end
-
-      signin_spo_user
-      visit prison_handovers_path(prison)
     end
 
-    scenario 'sorts POMs alphabetically' do
-      click_link('POM')
+    context 'without the SPO role' do
+      before do
+        signin_pom_user
+        stub_poms(prison, [user])
+      end
 
-      check_poms.each_with_index do |name, index|
-        expect(page).to have_css(".offender_row_#{index}", text: name)
+      it 'stops staff without the SPO role from viewing the SPO page' do
+        visit prison_handovers_path(prison)
+        expect(page).to have_current_path('/401')
       end
 
-      click_link('POM')
-      check_poms.reverse.each_with_index do |name, index|
-        expect(page).to have_css(".offender_row_#{index}", text: name)
+      it 'can load the POM handovers page', :js do
```

Comment:
Looks like this feature test now covers both the SPO handover page and the POM handover page.

It might be worth making this explicit/official by renaming the spec file, and changing the name on the root `feature` block. Otherwise it's not obvious that this file also tests a POM feature without looking for it.

Reply:
It looks like there's also some (very minimal) coverage of the POM handovers feature in `pom_caseload_feature_spec.rb` – would it be more logical to have it together in one place?

I think our expected user journeys have changed slightly following recent design changes – so could be a case of these feature specs needing to 'catch up' with that.